bin/configs.js

```diff
@@ -48,26 +52,54 @@ export const commands = {
 export const templates = {
     basic: {
         name: "basic",
+        isUrl: false,
```

Comment:
can we add a docstring above `templates` declaration indicating fields and it's purpose.
It'll really help future work.